Patches/InputTweaks.h

```diff
@@ -231,4 +231,214 @@ class KeyBindsHandler
 	BYTE GetPauseButtonBind();
 };
 
-extern InputTweaks InputTweaksRef;
+// Hitboxes for pause and memo menu
+class Hitboxes
+{
+private:
+	int top;
+	int left;
+	int height;	
+	int width;
+	int rows;
+	int columns;
+
+	float AspectRatio;
+	float ConstantAspectRatio = 16.f / 9.f;
+
+	int GetNormalizedHorizontal(int size)
+	{
+#pragma warning(disable : 4244)
+		return (size * AspectRatio) / ConstantAspectRatio;
+	}
+
+public:
+	Hitboxes(){}
+	Hitboxes(int top, int left, int height, int width, int rows, int columns, float AspectRatio)
+	{
+		this->AspectRatio = AspectRatio;
+
+		this->top = top;
+		this->left = GetNormalizedHorizontal(left);
+		this->height = height;
+		this->width = GetNormalizedHorizontal(width);
+		this->rows = rows;
+		this->columns = columns;
+	}
+	Hitboxes(int top, int left, int height, int width, int rows, int columns)
+	{
+		this->AspectRatio = 16.f / 9.f;
```

Comment:
Why not compute the AspectRatio using the width and the height rather than hard coding it as 16/9?  What happens if you are using other aspect ratios besides 16/9?  Will this cause an issue?